packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,209 @@
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: Textarea,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
+    focus: {
+      description: 'Focus',
+      control: 'boolean',
+    },
+    focusVisible: {
+      description: 'focusVisible',
+      control: 'boolean',
+    },
+    spellCheck: {
+      description: 'Spellcheck',
+      control: { type: 'select' },
+      options: ['', 'false', 'true'],
+    },
+  },
+  args: {
+    dir: 'auto',
+    disabled: false,
+    focus: false,
+    focusVisible: false,
+    hover: false,
+    invalid: false,
+    readOnly: false,
+    required: false,
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  tags: ['autodocs'],
+  parameters: {
+    status: {
+      type: 'WORK IN PROGRESS',
+    },
+    tokensPrefix: 'utrecht-textarea',
+    tokens,
+    tokensDefinition,
+    docs: {
+      description: {
+        component: readme,
+      },
+    },
+  },
+} satisfies Meta<typeof Textarea>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: `Styling via the \`.utrecht-textarea\` class name.`,
+      },
+    },
+  },
+};
+
+export const DefaultValue: Story = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => {
+    return <Textarea defaultValue={args.children} />;
+  },
+};
+
+export const PlaceholderRightToLeft: Story = {
+  name: 'Placeholder (right-to-left)',
+  args: {
+    ...Default.args,
+    placeholder: 'اكتب رسالتك',
+    children: '',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const RightToLeft: Story = {
+  args: {
+    ...Default.args,
+    children:
+      'لكن لا بد أن أوضح لك أن كل هذه الأفكار المغلوطة حول استنكار  النشوة وتمجيد الألم نشأت بالفعل، وسأعرض لك التفاصيل لتكتشف حقيقة وأساس تلك السعادة البشرية، فلا أحد يرفض أو يكره أو يتجنب الشعور بالسعادة، ولكن بفضل هؤلاء الأشخاص الذين لا يدركون بأن السعادة لا بد أن نستشعرها بصورة أكثر عقلانية ومنطقية فيعرضهم هذا لمواجهة الظروف الأليمة، وأكرر بأنه لا يوجد من يرغب في الحب ونيل المنال ويتلذذ بالآلام، الألم هو الألم ولكن نتيجة لظروف ما قد تكمن السعاده فيما نتحمله من كد وأسي.',
+  },
+  decorators: [arabicDecorator],
+};
+
+export const Spellcheck: Story = {
+  args: {
+    ...Default.args,
+    spellCheck: true,
```

Comment:
Since `spellCheck: true` is the default, I think the story should showcase `spellCheck: false`, otherwise there is nothing to see.